Gemfile.lock

```diff
@@ -261,14 +261,14 @@ GEM
       rspec-mocks (~> 3.4.0)
       rspec-support (~> 3.4.0)
     rspec-support (3.4.1)
-    rubocop (0.37.2)
-      parser (>= 2.3.0.4, < 3.0)
+    rubocop (0.40.0)
```

Comment:
@chriskacerguis I also agree with @Fryguy about carefully upgrading rubocop and haml_lint, let's leave that out of this PR.